app/src/main/res/values/strings.xml

```diff
@@ -40,4 +40,16 @@
     <string name="title_history">History</string>
     <string name="title_insights">Insights</string>
     <string name="title_profile">Profile</string>
+
+    <string name="tracking_tab_title">Tracking</string>
+    <string name="measuring_tab_title">Measuring</string>
+
+    <string name="study_burst_progress_bar_day">Day</string>
+    <string name="study_burst_progress_bar_title">Study Burst</string>
+    <string name="measuring_center_label">WALK AND BALANCE</string>
+    <string name="measuring_left_label">FINGER TAPPING</string>
+    <string name="measuring_right_label">TREMOR\nTEST</string>
```

Comment:
Are these line breaks intentional? I think allow the TextView to size itself and wrap its own text is better. I think maxlines=2 should do it

Reply:
The line break is intentional. Otherwise the label likes to put it all on one line, which contradicts the design

Reply:
Right. I'm saying the content (string) and presentation (capitalization, line breaks) should be treated as separate concerns. 

Thinking of strings as content, when doing i18n translations should "FINGER TAPPING" and "Finger Tapping" be translated differently? Would "TREMOR TEST" in Mandarin need a line break? 

Thinking of the presentation of the strings, I think the wrapping is an artifact of of screen real estate. In a responsive design, "WALK AND BALANCE" may take up 3 lines on a narrower screen, and be on one line on a wider screen.